### ISSUE TITLE
Image overlays don't seem to work properly...

### ISSUE THREAD

**Issue by GarThor** [OfficerHalf/homebrewery-vscode#21]
![image](https://user-images.githubusercontent.com/43391503/89961514-bac2fc80-dbf6-11ea-8e1e-ddabc29c33b8.png)

This feature is documented in a few different reddit posts I found...

https://www.reddit.com/r/UnearthedArcana/comments/6a469d/the_homebrewery_images_and_blending/

https://www.reddit.com/r/UnearthedArcana/comments/5nw1ou/watercolor_your_own_images_in_homebrewery/

https://www.reddit.com/r/UnearthedArcana/comments/656l98/formatting_tips_for_the_homebrewery/

https://www.reddit.com/r/UnearthedArcana/comments/6q81jx/homebrewery_formatting_guide/?utm_source=BD&utm_medium=Search&utm_name=Bing&utm_content=PSR1

**Comment by GarThor** [comment]
hmm... wait... I think I figured it out... the image was positioned way off of the document somehow...

Still not generating the same results though...

![image](https://user-images.githubusercontent.com/43391503/89962257-0bd3f000-dbf9-11ea-9148-5bd940571f39.png)

**Comment by nathonius** [comment]
Yeah I'm not sure how custom css reacts, for reasons I cannot fathom there are subtle differences between how the homebrewery and vs code render. I haven't updated this in a while, it could also be that there's been an update to the homebrewery css. Will update it on this package in a day or two.

**Comment by GarThor** [comment]
Man, its been a while since I've done any HTML... this image blending is kinda mindblowing... I really wish there was an easier and more uniform way to make it happen.  It'd be really neat to scrub the edges of that map a bit to get a more "worn" look without having to photoshop the hell out of it... >_<

**Comment by GarThor** [comment]
Mix/Blend mode doesn't seem to work either... >_>

![image](https://user-images.githubusercontent.com/43391503/89973991-ad1d6f00-dc16-11ea-99b4-a73e81945824.png)

seems to work fine if I put the style in a div tag around the <img> tag though? weird... -_-
